client/src/components/Dashboard/Dashboard.styled.js

```diff
@@ -35,6 +36,8 @@ export const SidebarOption = styled.div`
 	border-radius: 6px;
 	transition: all 200ms;
 
+	${({ active }) => active && css`{background-color: ${lighten(0.4, $grey)}`}
```

Comment:
Is the CSS helper needed? Couldn't this just be:
```javascript
${({ active }) => active && `background-color: ${lighten(0.4, $grey)}`};
```

Reply:
Nice catch, I've actually had more properties there but in the end only `background-color` remained. I'll fix this in next PR.

Reply:
I actually made a small commit to remove it before merging